src/giskardpy/tfwrapper.py

```diff
@@ -31,6 +32,24 @@ def init(tf_buffer_size=15):
     rospy.sleep(5.0)
 
 
+def get_full_frame_name(frame_name):
+    """
+    Gets the full tf frame name if the frame with the name frame_name
+    is in a separate namespace.
+
+    :rtype: str
+    """
+    global tfBuffer
+    tf_frames = tfBuffer._getFrameStrings()
+    for tf_frame in tf_frames:
+        try:
+            frame = tf_frame[tf_frame.index("/") + 1:]
+            if frame == frame_name:
+                return tf_frame
+        except ValueError:
+            continue
```

Comment:
What happens if `frame_name` is already the full frame name? I think this function would return `None`, which is not correct. 
If you don't find a match, you should return a `KeyError`.

Reply:
It would probably be a good idea to add a test_tf_wrapper file with some unit tests for your new functions.